asciidoc/courses/aura-dashboards/modules/2-understand-your-data/lessons/1-describe-data-model/lesson.adoc

```diff
@@ -0,0 +1,132 @@
+= Describe your movie data model
+:type: lesson
+:order: 1
+
+In this lesson, you will learn how to describe your movie data model in Neo4j Aura and design dashboards that align with stakeholder requirements. This includes understanding the structure of your project, the relationships between different components, and the properties that define them.
+
+By the end of this lesson, you should be able to:
+
+* Identify the key entities in your project
+* Describe the relationships between those entities
+* List the properties associated with each entity and relationship
```

Comment:
Technically, a relationship is also an _entity_ in a graph db. It would be more accurate to say _node_.